Update conversation api (#4546)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/conversation/howto-conversation-layer.md

{{< /tabs >}}

## Advanced features

The conversation API supports the following features:

> 1. Prompt caching - Allows developers to cache prompts in Dapr, leading to much faster response times and saving costs on egress and associated costs of inserting the prompt into the LLM provider's cache.

Reviewer: ```suggestion
1. **Prompt caching:** Allows developers to cache prompts in Dapr, leading to much faster response times and reducing costs on egress and on inserting the prompt into the LLM provider's cache.
```

> 2. PII scrub - Allows for the obfuscation of data going in and out of the LLM.

Reviewer: ```suggestion
1. **PII scrubbing:** Allows for the obfuscation of data going in and out of the LLM.
```

> To learn more on how to enable these features, see the API reference({{< ref conversation_api.md >}}) page.

Reviewer: ```suggestion
To learn how to enable these features, see the [conversation API reference guide]({{< ref conversation_api.md >}}).
```
